runtime/Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG SPRING_ZEEBE_VERSION=8.1.4
+FROM eclipse-temurin:17.0.5_8-jre
+
+RUN mkdir /opt/app
+# Download connectors from maven central
+ADD https://repo1.maven.org/maven2/io/camunda/spring-zeebe-connector-runtime/${AWS_LAMBDA_VERSION}/spring-zeebe-connector-runtime-${SPRING_ZEEBE_VERSION}.jar /opt/app/
```

Comment:
```suggestion
ADD https://repo1.maven.org/maven2/io/camunda/spring-zeebe-connector-runtime/${SPRING_ZEEBE_VERSION}/spring-zeebe-connector-runtime-${SPRING_ZEEBE_VERSION}.jar /opt/app/
```

Reply:
it does not have the `with-depencies` suffix - there is only one jar built in the runtime module. The version makes a lot of sense though 🙈

Reply:
But the Connector runtime JAR we create should have all dependencies included using Maven Shade 🤔

Reply:
Let me look into this

Reply:
![grafik](https://user-images.githubusercontent.com/1217596/201313079-61810e5d-7600-45dd-80b8-4e2d793bafb4.png)

The shaded version has the normal artifact name

Reply:
https://github.com/camunda-community-hub/spring-zeebe/pull/260

Reply:
Why do I not get a shaded artifact? 🤔

Reply:
What do you run to get those artifacts?

Reply:
Use the latest master version - there was something forgotten ;-)
https://github.com/camunda-community-hub/spring-zeebe/blob/master/connector-runtime/pom.xml#L100